### ISSUE TITLE
Use templated ignore globs based on language detected

### ISSUE THREAD

**Issue by satwikkansal** [coala/coala-quickstart#136]
Similar to the way github suggests `.gitignore`, we can either create our own templates or use the Github's API. 

Links
https://developer.github.com/v3/gitignore/
https://github.com/github/gitignore

status/needs design

**Comment by satwikkansal** [comment]
Agree, we can add a prompt if the user would like to ignore files as recommended by Github's standard, if they respond positively, we generate the coafile with those ignore globs.

**Comment by jayvdb** [comment]
We could ask if the want to generate a `.gitignore` , or a .ignore file, etc.

If they dont want any, they are not helpable...

**Comment by satwikkansal** [comment]
Yeah, it'd be helpful in generating more specific `.coafile` omitting the standard kind of files that users wanted to ignore but were lazy to create a `.gitignore` or write them down.
